### PR TITLE
Security and sanitization patches

### PR DESCRIPTION
**Pull request by bradly-shorthand**
Front: https://app.frontapp.com/open/cnv_e3qqnrh?key=poa6Uar6EVXPytcPyyJNmV0ieyK49zlq

### CODE REVIEW

File: src/templates/single-shorthand_story.php [hunk 1]
```diff
@@ -1,17 +1,33 @@
...
 	if (post_password_required($post->ID)) {
-		 echo get_the_password_form();
+		 echo wp_kses_post(get_the_password_form());
 	} else {
 		while ( have_posts() ) : the_post();
 			$meta = get_post_meta($post->ID);
+
+
+		// Get the content, but don't output it yet
+		$story_body = trim($meta['story_body'][0]);
```
Comment (james-shorthand) [inline src/templates/single-shorthand_story.php]: Discussed via call this function is stripping out data: on base64 images, @bradly-shorthand is going to take a look at it.

File: src/shorthand_connect.php [hunk 2]
```diff
@@ -518,8 +551,8 @@ function shand_post_type_tags_fix($request)
 /* Activation Hook */
 function shand_shorthand_activate()
 {
-	shand_create_post_type();
 	flush_rewrite_rules();
+	shand_create_post_type();
```
Comment (james-shorthand) [inline src/shorthand_connect.php:555]: As discussed, via Slack this will need to be placed above the `flush_rewrite_rules();` function.

**Review by ckortekaas — APPROVED** [review]
I've read through the changes in code and they made sense to me. Thanks for doing this. After reading the Front thread I'm guessing this tackles at least some of their concerns.

**Comment by bradly-shorthand** [comment]
This is on hold, there's a new bug with the header images which is proving to be quite a pain to resolve without encountering other issues

**Review by james-shorthand — APPROVED** [review]
Happy with this now, everything seems to be working as expected after a few final tweaks.

@bradly-shorthand can you please verify everything looks okay from your testing and we will get this released today?

**Comment by bradly-shorthand** [comment]
All tested and happy.

**Pull request merged**